auto/modules/wasm

```diff
@@ -0,0 +1,189 @@
+# Copyright (C) NGINX, Inc.
+
+
+NXT_WASM_RUNTIME=wasmtime
+
+shift
+
+for nxt_option; do
+
+    case "$nxt_option" in
+        -*=*) value=`echo "$nxt_option" | sed -e 's/[-_a-zA-Z0-9]*=//'`     ;;
+           *) value="" ;;
+    esac
+
+    case "$nxt_option" in
+
+        --runtime=*)		NXT_WASM_RUNTIME="$value"		    ;;
+        --module=*)		NXT_WASM_MODULE="$value"                    ;;
+        --include-path=*)	NXT_WASM_INCLUDE="$value"		    ;;
+        --lib-path=*)		NXT_WASM_LIB="$value"                       ;;
+
+        --help)
+            cat << END
+
+    --runtime			set the WASM runtime to use (default: wasmtime)
+    --module=NAME		set Unit WASM module name
+    --include-path=DIRECTORY	set directory path to wasmtime includes
+    --lib-path=DIRECTORY 	set directory path to libwasmtime.so library
+
+END
+            exit 0
+        ;;
+
+        *)
+            echo
+            echo $0: error: invalid wasm option \"$nxt_option\"
+            echo
+            exit 1
+        ;;
+    esac
+
+done
+
+
+if [ ! -f $NXT_AUTOCONF_DATA ]; then
+   echo
+   echo Please run common $0 before configuring module \"$nxt_module\".
+   echo
+   exit 1
+fi
+
+. $NXT_AUTOCONF_DATA
+
+NXT_WASM=wasm
+NXT_WASM_MODULE=${NXT_WASM_MODULE=${NXT_WASM##*/}}
+
+NXT_WASM_INCLUDE=${NXT_WASM_INCLUDE=}
+NXT_WASM_LIB=${NXT_WASM_LIB=}
+NXT_WASM_LDFLAGS=
+if [ "$NXT_WASM_RUNTIME" == "wasmtime" ]; then
```

Comment:
There are 3 of these `==` bashisms (lines 60, 76, 126) that should be a single `=` for Bourne Shell compatibility.

Reply:
Fixed!